### PR TITLE
LRT-4058: Fix missed PR comments

### PR DESCRIPTION
**Pull request by nikolaydanekov**

### CODE REVIEW

File: lendage-chat-service-main/src/main/java/com/lendage/chat/controller/v1/dto/ma/MortgageAdvisorListDto.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class MortgageAdvisorListDto {
+
+  private List<MortgageAdvisorDto> mortgageAdvisorDtos;
```
Comment (EugenShablovsky) [inline lendage-chat-service-main/src/main/java/com/lendage/chat/controller/v1/dto/ma/MortgageAdvisorListDto.java:11]: mortgageAdvisors

File: lendage-chat-service-main/src/main/java/com/lendage/chat/exception/MortgageAdvisorNotFoundException.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.lendage.chat.exception;
+
+import com.lendage.commons.exception.AppException;
+import org.springframework.http.HttpStatus;
+
+public class MortgageAdvisorNotFoundException extends AppException {
+
+  public MortgageAdvisorNotFoundException(String message) {
```
Comment (EugenShablovsky) [inline lendage-chat-service-main/src/main/java/com/lendage/chat/exception/MortgageAdvisorNotFoundException.java:8]: i believe message also can be static

**Review by Ivan-Tashev — APPROVED** [review]

File: migration.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ java -Dlogback.configurationFile=logback-spring.xml -cp "./app_dir/BOOT-INF/lib/
  --logLevel=WARNING \
  --driver=org.postgresql.Driver \
  --changeLogFile="db/liquibase-changelog.xml" \
- --url=jdbc:postgresql://lendage-cloudsql-proxy-np:80/lendage-communication-service-db \
+ --url=jdbc:postgresql://lendage-cloudsql-proxy-np:80/lendage-chat-service-db \
```
Comment (EugenShablovsky) [inline migration.sh:13]: application.properties contains dynamic parameter 
spring.datasource.url=jdbc:postgresql://localhost:5432/${DATABASE_NAME}

Comment (nikolaydanekov) [reply migration.sh:13]: I tried it locally but could not resolve the name that way. Also I saw that it was hardcoded across the other services so I thought it was intended.

**Review by EugenShablovsky — APPROVED** [review]

**Pull request merged**